### ISSUE TITLE
Filesystem metrics are not showing up with the node exporter integration

### ISSUE THREAD

**Issue by Dileep-Dora** [grafana/agent#150]
We're running cloud agent to remote write to Prometheus and using the node_exporter integration, This is the config
```server:
  log_level: error
  http_listen_port: 12345
prometheus:
  wal_directory: /data/wal
  global:
    scrape_interval: 5s
    external_labels:
      region: us-east-1
  configs:
    - name: agent
      host_filter: false
      scrape_configs:
        - job_name: elasticsearch
          metrics_path: /metrics
          static_configs:
          - targets:
            - localhost:9114
      remote_write:
        - url: <url>
          bearer_token: <token>
integrations:
  agent:
    enabled: false
  node_exporter:
    enabled: true

  prometheus_remote_write:
    - url: <url>
      bearer_token: <token>
```
But while using the dashboard found the filesystem metrics are not showing up, sample metrics expected are

- node_filesystem_avail_bytes, node_filesystem_size_bytes etc.

**Comment by rfratto** [comment]
Hi, thanks for reporting! This is a [known issue](https://github.com/grafana/agent/issues/127) which we've fixed but haven't included in a release yet. 

I'm planning on putting out a release today which will include the fix for this. If you don't want to wait or upgrade, you can workaround the issue by manually setting the default values for `filesystem_ignored_mount_points` and `filesystem_ignored_fs_types` in your `node_exporter` section like this:

```yaml
  node_exporter:
    enabled: true 
    filesystem_ignored_mount_points: "^/(dev|proc|sys|var/lib/docker/.+)($|/)"
    filesystem_ignored_fs_types: "^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$"
```

**Comment by Dileep-Dora** [comment]
Great!, we'll upgrade when you guys have a new release.

**Comment by rfratto** [comment]
We just released [v0.5.0](https://github.com/grafana/agent/releases/tag/v0.5.0) 🙂  I'm going to close this as fixed, but if you upgrade and are still seeing problems, feel free to comment and I can re-open.

**Comment by lherbeng** [comment]
I can't find the node_filesystem_avail_bytes and node_filesystem_size_bytes in my grafana dashboard. I am using grafana v10.1 and chart version kube-prometheus-stack-51.0.2 app version v0.68.0
